docs/usage-and-billing.mdx

```diff
@@ -38,7 +38,7 @@ All Semgrep Pro products are free under the **Team** tier for **10 monthly contr
 - Semgrep Cloud Platform
 - Semgrep Code
 - Semgrep Supply Chain
-- Semgrep Secrets (beta)
+- Semgrep Secrets
```

Comment:
```suggestion
```
Let's remove the entire line for now while we're in this weird in-between state!